src/components/VDialog/VDialog.js

```diff
@@ -171,9 +171,10 @@ export default {
     }, [dialog]))
 
     return h('div', {
-      'class': 'dialog__container',
+      staticClass: 'dialog__container',
       style: {
-        display: !this.$slots.activator && 'none' || this.fullWidth ? 'block' : 'inline-block'
+        // TODO: double-check this logic, it might need 'none'
+        display: (!this.$slots.activator || this.fullWidth) ? 'block' : 'inline-block'
```

Comment:
99c586cb056b71188d8b558426e34554c7b4d348

Reply:
reverted to original logic